### ISSUE TITLE
Redesign (officially v1)

### ISSUE THREAD

**Issue by akivajgordon** [akivajgordon/tikkun.io#22]
## TL;DR

Here is a mockup of the upcoming redesign:

![redesign](https://user-images.githubusercontent.com/6025036/41133998-3939e75e-6a98-11e8-818c-ad1208f455da.gif)

## Overview

The primary goal of this redesign is to help offer a true mobile experience. I felt that in order to do that, the UI needs to be as clean as possible, and it makes sense for it to be that way on the desktop experience as well. 

## To Do

These are the elements that implement this design:

* [ ] An accessible parsha picker on mobile
* [ ] Reasonable font-size on mobile
* [ ] Removing the next/previous page buttons in favor of two-way infinite scrolling. This has always been a dream of mine in this app, but I was never able to get the scrolling in reverse to feel natural and not jittery. B"H, I have solved that with a crafty approach, so I'm excited to get this in. There is a potential performance issue when toggling the annotations if there are a ton of loaded pages in the DOM, but there are techniques to solve that, too.
* [ ] Removing the "chrome" and keeping the UI really simple and clean

## Regressions

After implementing this design, here are some acceptable regressions, which I'd hope to tackle in the future:

* [ ] The "current parsha" indicator, may not reflect the current visible position, but rather would reflect the last selection from the parsha picker
* [ ] The last position will not be remembered from session to session, nor will the current state of the "show annotations" toggle.

## Technical changes

Currently, the app is built on Polymer v1, which was fun, but it's quite out of date now. I'm not a big fan of frameworks, so I don't feel like upgrading to Polymer v3, nor use React. I'm currently building this using [lit-html](https://github.com/Polymer/lit-html), which is pretty lightweight, but even then I think I want to just remove that and go vanilla JS. It's a pretty simple app that doesn't need all of this complication. I find myself fighting frameworks more than they end up benefiting me…

## Feedback

Thoughts about this redesign? I'm pretty deep into it at this point, but I'm always willing to hear feedback, so please feel free to share!
